File: Eva_3_prac_5.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author invitado
 */
public class PRINCIPAL {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        int ival = 5;
        //int iResu = calcularFactorial (ival);
        System.out.println("Resultado bro "+ calcularFactorial (ival));
    }
    public static int calcularFactorial(int ival){
        int iResu = 1;
        for (int i = ival; i >= 1; i--) {
            iResu *= i; // iResu = iResu * i;
            
            
        }
        return iResu;
    }
}

File: Eva_3_prac_5.cpp
/******************************************************************************

                              Online C++ Compiler.
               Code, Compile, Run and Debug C++ program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/

#include <iostream>

using namespace std;
int calcularFactorial (int ival){
    int iresu = 1;
     for (int i = ival; i >= 1; i--) {
            iresu *= i;
     }
     return iresu;
}
int main()
{
    cout<<"Resultado = " << calcularFactorial(5);

    return 0;
}
